.github/workflows/ci-goreleaser.yaml

```diff
@@ -20,6 +20,7 @@ jobs:
     strategy:
       matrix:
         GOOS: [linux, windows, darwin]
+        GOARCH: ["386", amd64, arm64, ppc64le]
```

Comment:
We ignore a couple combinations in the code that generates the goreleaser configuration https://github.com/open-telemetry/opentelemetry-collector-releases/blob/3971091c01a366e0b33df39a105daa2f38bef44d/cmd/goreleaser/internal/configure.go#L72-L75

Should we also ignore them in the matrix definition here?

Reply:
those end up getting ignored here: https://github.com/open-telemetry/opentelemetry-collector-releases/blob/49c07b47e3154a01604622a40256c4e4b0d33de8/.goreleaser.yaml#L37-L41, but i can add an ignore to the matrix definition

Reply:
excluded 👍🏻